include/HubFramework/HUBViewController.h

```diff
@@ -152,6 +152,14 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (BOOL)viewControllerShouldIgnoreHeaderComponentContentInset:(HUBViewController *)viewController;
 
+/**
+ *  Sent to a Hub Framework view controller's delegate to ask if content inset for top bar
+ *  (navigation bar and status bar) should be ignored
+ *
+ *  @param viewController The view controller which displays a header component
+ */
+- (BOOL)viewControllerShouldIgnoreTopBarContentInset:(HUBViewController *)viewController;
```

Comment:
Could this be merged into the API that was previously introduced for header components, and be something like `viewControllerShouldAutomaticallyManageTopContentInset:`?